src/core/mod.rs

```diff
@@ -292,16 +290,38 @@ pub const STACKS_EPOCHS_MAINNET: &[StacksEpoch] = &[
         epoch_id: StacksEpochId::Epoch10,
         start_height: 0,
         end_height: BITCOIN_MAINNET_FIRST_BLOCK_HEIGHT,
+        block_limit: ExecutionCost {
+            write_length: 15_000_000,
+            write_count: 7_750,
+            read_length: 100_000_000,
+            read_count: 7_750,
+            runtime: 5_000_000_000,
+        },
     },
     StacksEpoch {
         epoch_id: StacksEpochId::Epoch20,
         start_height: BITCOIN_MAINNET_FIRST_BLOCK_HEIGHT,
         end_height: STACKS_2_0_LAST_BLOCK_TO_PROCESS + 1,
+        block_limit: ExecutionCost {
+            write_length: 15_000_000,
+            write_count: 7_750,
+            read_length: 100_000_000,
+            read_count: 7_750,
+            runtime: 5_000_000_000,
+        },
     },
     StacksEpoch {
         epoch_id: StacksEpochId::Epoch2_05,
         start_height: STACKS_2_0_LAST_BLOCK_TO_PROCESS + 1,
         end_height: STACKS_EPOCH_MAX,
+        // TODO: Fill this in with the determined 2.05 costs.
```

Comment:
Don't we already know what the limits will be?  They're specified in SIP 012.

Reply:
Yes -- the block limits in SIP-012 are more or less finalized -- there's no new benchmarking happening that would impact the block limit definition there.

Reply:
updated

Reply:
Updated as of 340985579